From 8c8521600f2654e9840611bc202d2ef57400dea0 Mon Sep 17 00:00:00 2001
From: Yannick Fertre <yannick.fertre@foss.st.com>
Date: Tue, 11 Feb 2025 14:15:17 +0100
Subject: [PATCH] arm64: dts: st: add panel power on stm32mp257f-ev1 board

Lvds panel driver support the management of power supply.

Change-Id: I18e5cdbfd99e972b7d6ec69f10f162cb85d9ec1d
Signed-off-by: Yannick Fertre <yannick.fertre@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/436220
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 arch/arm64/boot/dts/st/stm32mp257f-ev1.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
index a8efce0eb459..2e4fc0104a03 100644
--- a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
@@ -146,6 +146,7 @@ panel_lvds: panel-lvds {
 		compatible = "edt,etml0700z9ndha", "panel-lvds";
 		enable-gpios = <&gpiog 15 GPIO_ACTIVE_HIGH>;
 		backlight = <&panel_lvds_backlight>;
+		power-supply = <&scmi_v3v3>;
 		default-on;
 		status = "okay";
 
-- 
2.50.1

